va/http.go

```diff
@@ -397,12 +347,39 @@ func (va *ValidationAuthorityImpl) processHTTPValidation(
 		return nil, nil, err
 	}
 
+	// Create an initial GET Request
+	initialURL := fmt.Sprintf("http://%s%s", host, path)
```

Comment:
Instead of Sprintf, let's build a `url.URL` and use its `.String()` function.

Reply:
Good suggestion, thank you. Implemented in 3d7b2b8